data/sql/updates/pending_db_world/rev_1730520438291701981.sql

```diff
@@ -0,0 +1,17 @@
+DELETE FROM creature_formations WHERE leaderGUID = 42583;
```

Comment:
Please group both deletes in the same query
Please use backticks around table names and colums 

```SQL
DELETE FROM `creature_formations` WHERE `leaderGUID` = 42583;
```